taskRequests/details/script.js

```diff
@@ -294,27 +319,103 @@ async function fetchTaskRequest() {
   data.approvedTo = approvedTo;
   return data;
 }
-const renderRejectButton = (taskRequest) => {
-  if (taskRequest?.status !== 'PENDING') {
-    rejectButton.disabled = true;
-  }
 
-  rejectButton.addEventListener('click', async () => {
-    const res = await updateTaskRequest(TaskRequestAction.REJECT);
-    if (res?.ok) {
-      rejectButton.disabled = true;
-    }
+const renderGithubIssue = async () => {
+  converter = new showdown.Converter({
+    tables: true,
+    simplifiedAutoLink: true,
+    tasklists: true,
+    simplifiedAutoLink: true,
+    ghCodeBlocks: true,
+    openLinksInNewWindow: true,
   });
+  let res = await fetch(taskRequest?.externalIssueUrl);
+  res = await res.json();
+  taskSkeleton.classList.add('hidden');
+  taskContainer.classList.add('task__issue__container');
+  taskContainer.append(
+    createCustomElement({
+      tagName: 'h2',
+      innerHTML: res?.title,
+      id: 'issue_title',
+    }),
+  );
+  taskContainer.appendChild(
+    createCustomElement({
+      tagName: 'p',
+      id: 'issue_time_author',
+      innerHTML:
+        'Opened on ' +
+        new Date(res?.created_at).toDateString() +
+        ' by <a class="card__link" href="' +
+        res?.user?.html_url +
+        '">' +
+        res?.user?.login +
+        '</a>',
```

Comment:
Since rest of the pages uses methods like `createCustomElement` to generate html content. Should we use the same here to maintain consistency?

Reply:
Implemented createCustomElement.